examples/Arithmetic/main.roc

```diff
@@ -0,0 +1,65 @@
+app "example"
+    packages { pf: "https://github.com/roc-lang/basic-cli/releases/download/0.3.1/97mY3sUwo433-pcnEQUlMhn-sWiIf_J9bPhcAFZoqY4.tar.br" }
+    imports [
+        pf.Stdout,
+        pf.Task.{ await },
+        pf.Arg,
+    ]
+    provides [main] to pf
+
+TaskErrors : [InvalidArg, InvalidNumStr]
+
+main =
+    task =
+        args <- readArgs |> await
+
+        calculate = \(operation, result) ->
+            resultStr = Num.toStr result
+
+            "\(operation): \(resultStr)"
+
+        results =
+            [
+                ("sum", args.a + args.b),
+                ("difference", args.a - args.b),
+                ("product", args.a * args.b),
+                ("integer quotient", args.a // args.b),
+                ("remainder", args.a % args.b),
+                ("exponentiation", Num.powInt args.a args.b),
+            ]
+            |> List.map calculate
```

Comment:
I think we should split out the vital part of the solution into a separate function. Readers are likely to just want to copy that function.